### DISCUSSION TITLE
Portland light rail lines

### DISCUSSION THREAD

**Discussion by SIVAH-AKASH** [organicmaps/organicmaps#4244] [Q&A]
Hi everyone,

I'm noticing that Portland, OR, USA has all the light rail stations in OSM but is missing the lines/tracks themselves. Is there an easy way to map them out? I'm thinking of something as simple as StreetComplete. I'm not software literate but am willing to do physical mapping work if needed.

Thanks!

**Comment by biodranik** [comment]
Please check https://cdn.organicmaps.app/subway/ and https://docs.google.com/spreadsheets/d/1SEW1-NiNOnA2qDwievcxYV1FOaQl1mb1fdeyqAxHu3k/edit?usp=sharing
